src/store/modules/drawing.store.js

```diff
@@ -69,5 +126,11 @@ export default {
         deleteDrawingFeature: (state, { featureId }) =>
             (state.featureIds = state.featureIds.filter((featId) => featId !== featureId)),
         setDrawingFeatures: (state, { featureIds }) => (state.featureIds = featureIds),
+        setShowDrawingOverlay(state, { show, online, kmlId, title }) {
+            state.drawingOverlay.show = show
+            state.drawingOverlay.title = title
+            state.drawingOverlay.online = online
+            state.drawingOverlay.kmlId = kmlId
```

Comment:
there two do not match what you have described above in the state, it should be 
```
state.online = online
state.temporaryKmlId = kmlId
```
from what I understand